### PR TITLE
Creating new RHEL 8 STIG GUI profile

### PR DESCRIPTION
**Pull request by carlosmmatos**
#### Description:

New RHEL 8 STIG with GUI profile. Tackles issues that allow existing customers to utilize the RHEL 8 STIG during install, and still maintain a graphical env if they chose to have one.

### CODE REVIEW

File: rhel8/profiles/stig_gui.profile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    SMEs:
+        - carlosmmatos
+
+reference: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux
+
+title: 'DISA STIG with GUI for Red Hat Enterprise Linux 8'
+
+description: |-
+    This profile contains configuration checks that align to the
+    DISA STIG with GUI for Red Hat Enterprise Linux 8.
```
Comment (ggbecker) [inline rhel8/profiles/stig_gui.profile:14]: I think we have missed the versioning for RHEL8 STIG profile. We should have `V1R1` stated in this line.
```suggestion
    DISA STIG with GUI for Red Hat Enterprise Linux 8 V1R1.
```

Would you care to change in the original stig.profile as well? On the other hand V1R2 is about to be released.

Comment (carlosmmatos) [reply rhel8/profiles/stig_gui.profile:14]: Roger that.

**Review by yuumasato — APPROVED** [review]
LGTM.
But I leave the final ack to @ggbecker

**Review by ggbecker — APPROVED** [review]

**Pull request merged**